### DISCUSSION TITLE
Integration

### DISCUSSION THREAD

**Discussion by lolokeer** [ParadoxGameConverters/ImperatorToCK3#1951] [Ideas]
Integrate Reanimāta and Full Mechanical Overhaul to conversion

**Comment by IhateTrains** [comment]
Hi, FMO is already supported alongside a few other Invictus submods, full list here:
https://github.com/ParadoxGameConverters/ImperatorToCK3/blob/9faa867971256904ab48271cf3b1da0f7ff8e13d/ImperatorToCK3/Data_Files/converter_globals/FAQ.txt#L15

Additional mod support is up to the volounteers, so if you want Reanimāta to be supported, you need to provide mappings (for instance, for the new cultures Reanimāta adds) yourself, or find someone else to do it.
